models/repo_list_test.go

```diff
@@ -74,6 +74,34 @@ func TestSearchRepositoryByName(t *testing.T) {
 	assert.Empty(t, repos)
 	assert.Equal(t, int64(0), count)
 
+	// Test search within description
+	repos, count, err = SearchRepository(&SearchRepoOptions{
```

Comment:
Please update the test function name to reflect the new name `func TestSearchRepository(t *testing.T)` (even if it still SearchRepositoryByName the real func tested is SearchRepository)

Reply:
Fixed